exception.username.already.exists=Имя пользователя уже существует
exception.username.already.exists.detail=Введенное вами имя пользователя уже используется. Пожалуйста, попробуйте другое.
exception.not.found=Не найдено
exception.not.found.detail={0} не найдено